tests/testthat/test-methods.R

```diff
@@ -70,21 +69,16 @@ test_that("as.data.frame.lints", {
 })
 
 test_that("summary.lints() works (no lints)", {
-  no_lints <- lint(
-    "x <- 1\n",
-    linters = assignment_linter()
-  )
+  no_lints <- lint("x <- 1\n", linters = assignment_linter())
```

Comment:
This `lint(text = "x = 1", linters = assignment_linter())` is repeated a lot. Should we:

 - Have a helper in helpers.R: `run_assignment_linter()` (perhaps with argument `should_lint = TRUE/FALSE`?
 - Just cache the output within this test file: `example_with_lint <- lint(...); example_without_lint <- lint(...)`
 - Neither?

Reply:
I would prefer to do neither. I learned from you the concept of keeping tests [DAMP](https://enterprisecraftsmanship.com/posts/dry-damp-unit-tests/), and I think that's relevant here. 

IMO, as they stand, the tests are more readable than they would be with this additional abstraction introduced.